package.json

```diff
@@ -1,108 +1,108 @@
-  "bugs": {
-    "url": "https://github.com/magento/pwa-studio/issues"
-  },
-  "scripts": {
-    "build": "yarn venia run build",
-    "build:dev": "yarn venia run build:dev",
-    "clean:all": "yarn workspaces run -s clean && rimraf ./node_modules",
-    "clean:dist": "yarn workspaces run clean",
-    "coveralls": "cat ./coverage/lcov.info | coveralls",
-    "danger": "danger-ci",
-    "lint": "eslint '@(packages|scripts)/**/{*.js,package.json}' --ignore-pattern node_modules --ignore-pattern storybook-dist",
-    "postbuild": "rimraf \"./packages/*/dist/{,**/}__*__\"",
-    "prepare": "node scripts/monorepo-introduction.js",
-    "prettier": "prettier --loglevel warn --write '@(packages|scripts)/**/*.@(css|graphql|js)' '*.js'",
-    "prettier:check": "prettier --list-different '@(packages|scripts)/**/*.@(css|graphql|js)' '*.js'",
-    "stage:venia": "yarn venia run start",
-    "stats:venia": "yarn venia run build:analyze && yarn venia run stats",
-    "storybook:venia": "yarn venia run storybook",
-    "test": "jest",
-    "test:ci": "jest --no-cache -i --json --outputFile=test-results.json",
-    "test:debug": "node --inspect-brk node_modules/.bin/jest -i --no-cache",
-    "test:dev": "jest --watch",
-    "validate-queries": "yarn venia run validate-queries",
-    "venia": "node ./packages/pwa-buildpack/bin/buildpack load-env ./packages/venia-concept && yarn workspace @magento/venia-concept",
-    "watch:all": "node scripts/watch-all.js",
-    "watch:venia": "yarn venia run watch"
-  },
-  "devDependencies": {
-    "@magento/eslint-config": "~1.4.1",
-    "@types/jest": "~24.0.18",
-    "caller-id": "~0.1.0",
-    "chalk": "~2.4.2",
-    "chokidar": "~2.1.2",
-    "coveralls": "~3.0.3",
-    "danger": "^7.1.4",
-    "eslint": "~5.14.1",
-    "eslint-plugin-babel": "~5.3.0",
-    "eslint-plugin-graphql": "~3.0.3",
-    "eslint-plugin-jsx-a11y": "~6.2.1",
-    "eslint-plugin-node": "~8.0.1",
-    "eslint-plugin-package-json": "~0.1.3",
-    "eslint-plugin-react": "~7.12.4",
-    "eslint-plugin-react-hooks": "~2.0.1",
-    "execa": "~1.0.0",
-    "figures": "~2.0.0",
-    "first-run": "~2.0.0",
-    "graphql-tag": "~2.10.1",
-    "husky": "~1.3.1",
-    "identity-obj-proxy": "~3.0.0",
-    "jest": "~24.3.1",
-    "jest-fetch-mock": "~2.1.1",
-    "jest-junit": "~6.3.0",
-    "jest-transform-graphql": "~2.1.0",
-    "lerna": "~3.13.0",
-    "lodash.debounce": "~4.0.8",
-    "prettier": "~1.16.4",
-    "rimraf": "~2.6.3"
-  },
-  "optionalDependencies": {
-    "bundlesize": "~0.17.1",
-    "sharp": "~0.22.1"
-  },
-  "resolutions": {
-    "graphql": "~14.3.1",
-    "**/graphql-cli/npm-run": "~5.0.0"
-  },
-  "engines": {
-    "node": ">=10.x",
-    "yarn": ">=1.12.0"
-  },
-  "husky": {
-    "hooks": {
-      "pre-push": "yarn run prettier:check && yarn run lint"
-    }
-  },
-  "bundlesize": [
-    {
-      "path": "./packages/venia-concept/dist/{client,vendors,sw}?(.)*.js",
-      "maxSize": "250 kB"
+    "name": "@magento/pwa-studio",
```

Comment:
Why is it changing indentation from two spaces to four? Something appears to be misconfigured on this branch.

Reply:
Handled.